core/process.py

```
@@ -6,12 +6,17 @@
 import time
 from typing import Any, Dict
 import json
+import threading
 
 from config.persona_utils import persona_manager
```

Ruff (0.11.9)

`config.persona_utils.persona_manager` imported but unused

Remove unused import: `config.persona_utils.persona_manager`

(F401)